data-prepper-plugins/cloudwatch-logs/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'java'
+    id 'java-library'
+}
+
+group = 'org.opensearch.dataprepper'
+version = '2.3.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    api project(':data-prepper-api')
+    implementation project(':data-prepper-plugins:aws-plugin-api')
+    implementation project(path: ':data-prepper-plugins:s3-sink')
```

Comment:
I think you are adding this to import AwsAuthenticationOptions. But we currently do not do like this. Ideally the `AwsAuthenticationOptions` should be moved to common directory. But for now, it is OK if you duplicate the code and put it in cloud watch logs directory

Reply:
Yeah, I realized that by adding that dependency we are not isolating the separate plugins. I have removed the dependency and proceeded to use my own version of AwsConfig.